### PR TITLE
Create CONTRIBUTING.md file

### PR DESCRIPTION
**Pull request by swastikbanerjee07**

### CODE REVIEW

**Comment by swastikbanerjee07** [comment]
Done.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+###### Pre-requisites:
+Install Git: 
+*Are you a first time user and just beginning with Open-Source projects? No problem! Here's [how you can install Git on your computer](https://www.digitalocean.com/community/tutorials/how-to-contribute-to-open-source-getting-started-with-git)*
+###### After you are done installing Git on your computer:
+1. Fork this repository on Github.
+2. Clone it onto your computer by opening terminal and running the command $ git clone https://github.com/{your username}/Data-Structures-And-Algorithms-Hacktoberfest18.git
```
Comment (matthewsamuel95) [inline CONTRIBUTING.md]: change this url

**Comment by matthewsamuel95** [comment]
any updates? this is much needed

**Comment by swastikbanerjee07** [comment]
I'm working on it. Will send a PR by today night. :)

**Comment by matthewsamuel95** [comment]
plz edit the readme.md and link ur contributing.md to it near the top

also dont forget the file structure how to follow

**Comment by swastikbanerjee07** [comment]
Updated contributing.md

**Comment by swastikbanerjee07** [comment]
Updated README and resolved conflicts.
Done.

**Pull request merged**